games-arcade/wormux/wormux-master.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="Very nice game"
+DESCRIPTION="Arcade game based on Worms"
+HOMEPAGE="http://wormux.org"
+COPYRIGHT="<a-team>"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/a-team/wormux/archive/master.zip"
+ARCHITECTURES="x86_64 x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
+PROVIDES="
+	wormux = $portVersion
+"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libsdl$secondaryArchSuffix
+	lib:libSDL_image$secondaryArchSuffix
+	lib:libSDL_mixer$secondaryArchSuffix
+	lib:libSDL_sound$secondaryArchSuffix
+	lib:libSDL_ttf$secondaryArchSuffix
+	lib:libxml2$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libSDL_gfx$secondaryArchSuffix
+	lib:libSDL_net$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	devel:libglu$secondaryArchSuffix
+	devel:libsdl$secondaryArchSuffix
+	devel:libsdl_image$secondaryArchSuffix
+	devel:libsdl_mixer$secondaryArchSuffix
+	devel:libsdl_sound$secondaryArchSuffix
+	devel:libsdl_ttf$secondaryArchSuffix
+	devel:libxml2$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libsdl_gfx$secondaryArchSuffix
+	devel:libsdl_net$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	haiku_devel
```

Comment:
This is the same as "haiku${secondaryArchSuffix}_devel". Unneeded, please remove this.